<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(82, 132, 255);</string>
  </property>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>16</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>471</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../OneDrive/Desktop/temp12.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>651</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_34">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtName">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter your full name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_35">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtEmail">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter your valid email id</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_36">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>Phone no:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtMobileno">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter your mobile number</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_37">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="txtCombobox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
      <item>
       <property name="text">
        <string>Select Your Gender</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Others</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_38">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
</string>
      </property>
      <property name="text">
       <string>DOB:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="txtDate">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_39">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;

</string>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="txtAddress">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter your current address</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_40">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;

</string>
      </property>
      <property name="text">
       <string>Post:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_41">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 9, 43);
font: 10pt &quot;Arial Rounded MT Bold&quot;;

</string>
      </property>
      <property name="text">
       <string>Salary:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="txtSalary">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter your current salary</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txtPost">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
margin: 4px;
padding: 6px;
border:none;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter your current post</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline;
color: rgb(31, 77, 186);
font: 25pt &quot;Arial Rounded MT Bold&quot;;
font: 25pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
border: 3px solid rgb(31, 77, 186);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>EMPLOYEE MANAGEMENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>16</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>680</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:  rgb(0, 9, 43);
font: 10pt &quot;Bell MT&quot;;
border: 1px groove rgb(0, 9, 43);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>440</y>
     <width>971</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../OneDrive/Desktop/tempedit.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnToggleTable">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>680</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:  rgb(0, 9, 43);
font: 10pt &quot;Bell MT&quot;;
border: 1px groove rgb(0, 9, 43);
</string>
   </property>
   <property name="text">
    <string>Show Table</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>591</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(0, 9, 43);
font: 75 8pt &quot;Arial&quot;;
border: 1px groove black;
font: 11pt;
border-radius: 6px;
color: rgb(19, 1, 44);
color: rgb(82, 132, 255);






</string>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DOB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
  </widget>
  <zorder>label_43</zorder>
  <zorder>label_3</zorder>
  <zorder>line_2</zorder>
  <zorder>formLayoutWidget_2</zorder>
  <zorder>label_42</zorder>
  <zorder>line</zorder>
  <zorder>btnDelete</zorder>
  <zorder>btnToggleTable</zorder>
  <zorder>tableWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
